xwiki-platform-core/xwiki-platform-user/xwiki-platform-user-profile/xwiki-platform-user-profile-ui/src/main/resources/XWiki/XWikiUserProfileSheet.xml

```diff
@@ -145,9 +150,12 @@ $services.localization.render('platform.core.profile.category.profile.disabled')
       #end
       &lt;div class='userRecentChanges'&gt;
       #if ($xcontext.user == $doc.fullName)
-        &lt;h1&gt;$services.localization.render('platform.core.profile.section.activity')&lt;/h1&gt;
+        &lt;h2&gt;$services.localization.render('platform.core.profile.section.activity')&lt;/h2&gt;
```

Comment:
Since you're modifying this line, would be good to also escape the XML from the heading text using ``$escapetool.xml()``.